cloud/passportauth/integration/UserSeedData.ts

```diff
@@ -0,0 +1,70 @@
+export interface User {
+  avatar: string;
+  banner: string;
+  email: string;
+  id: string;
+  name: string;
+  notes: string;
+  password: string;
+  phone: string;
+  position: string;
+  username: string;
+  roles: string[];
+}
+
+const userSeedData: User[] = [
+  {
+    avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/kolage/128.jpg',
```

Comment:
this should be in examples folder. Core module should not contain any dummy data